full/build.gradle

```diff
@@ -20,7 +19,7 @@ jar {
 }
 
 compileKotlin {
-    kotlinOptions.jvmTarget = "1.8"
+    kotlinOptions.jvmTarget = "17"
```

Comment:
Update this in the other branches

Reply:
Yes, but in the other branches 4.2, 4.3 and 4.4, I guess you should update it to 11 rather than 17.